samples/snippets/src/main/java/aiplatform/ImportDataImageClassificationSample.java

```diff
@@ -0,0 +1,88 @@
+import com.google.cloud.aiplatform.v1beta1.ImportDataOperationMetadata;
+import com.google.cloud.aiplatform.v1beta1.ImportDataResponse;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class ImportDataImageClassificationSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String datasetId = "YOUR_DATASET_ID";
+    String gcsSourceUri =
+        "gs://YOUR_GCS_SOURCE_BUCKET/path_to_your_image_source/[file.csv/file.jsonl]";
+    importDataImageClassificationSample(project, datasetId, gcsSourceUri);
+  }
+
+  static void importDataImageClassificationSample(
+      String project, String datasetId, String gcsSourceUri)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    DatasetServiceSettings datasetServiceSettings =
+        DatasetServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DatasetServiceClient datasetServiceClient =
+        DatasetServiceClient.create(datasetServiceSettings)) {
+      String location = "us-central1";
+      String importSchemaUri =
+          "gs://google-cloud-aiplatform/schema/dataset/ioformat/"
+                  + "image_classification_single_label_io_format_1.0.0.yaml";
+
+      GcsSource.Builder gcsSource = GcsSource.newBuilder();
+      gcsSource.addUris(gcsSourceUri);
+      DatasetName datasetName = DatasetName.of(project, location, datasetId);
+
+      List<ImportDataConfig> importDataConfigList =
+          Collections.singletonList(
+              ImportDataConfig.newBuilder()
+                  .setGcsSource(gcsSource)
+                  .setImportSchemaUri(importSchemaUri)
+                  .build());
+
+      OperationFuture<ImportDataResponse, ImportDataOperationMetadata> importDataResponseFuture =
+          datasetServiceClient.importDataAsync(datasetName, importDataConfigList);
+      System.out.format(
+          "Operation name: %s\n", importDataResponseFuture.getInitialFuture().get().getName());
+      System.out.println("Waiting for operation to finish...");
+      ImportDataResponse importDataResponse = importDataResponseFuture.get(300, TimeUnit.SECONDS);
+
+      System.out.format("Import Data Image Classification Response: %s\n", importDataResponse);
```

Comment:
Can you verify that you dont need to call the `toString` method here? Printing out the object signature is probably not very useful.